docs/reference/workloads-18-19.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Workloads API changes in version 1.8 and 1.9
+approvers:
+- steveperry-53
+- kow3ns
+---
+
+## Overview
+
+The Kubernetes core workload API consists of the Deployment, DaemonSet, ReplicaSet, and StatefulSet API objects. In order to provide a stable API for users to orchestrate their workloads, we are prioritizing promoting the core workload API to GA. The batch workload API (Job and CronJob), while also important, is not part of this effort, and it will have a separate path to GA stability.
```

Comment:
Workloads API is the larger concept per the reference docs. I recommend "Workloads API" but then further modified because: 
In v1.7, it includes three groups: extensions, apps, and batch. In 1.8, it includes two groups: apps and batch. The groups also rejoice in different versions. "core" here = extensions + apps, or apps, depending on the version. I don't know what to do about it, but I'm leaving "core" for now bc it helps distinguish between groups. IF you're already something of a K8s API insider, I know. But really this is an API design issue, not a docs issue, so while I agree that we need an authoritative convention, I guess I'm recommending here that we iterate toward said convention.